### PR TITLE
section commit for June 12th, '20

### PR DESCRIPTION
**Pull request by croqueGrec09**

### CODE REVIEW

File: app/grails-app/views/subscription/members.gsp [hunk 1]
```diff
@@ -189,7 +189,7 @@
                         <td></td>
                     </g:if>
                     <%
-                        LinkedHashMap<String, List> links = navigationGenerationService.generateNavigation(Subscription.class.name, sub.id)
+                        LinkedHashMap<String, List> links = linksGenerationService.generateNavigation(com.k_int.kbplus.GenericOIDService.getOID(sub))
```
Comment (Moetez16) [inline app/grails-app/views/subscription/members.gsp:192]: com.k_int.kbplus.

File: app/grails-app/views/subscription/_actions.gsp [hunk 2]
```diff
@@ -134,7 +134,7 @@
 
 
 
-            <g:set var="previousSubscriptions" value="${Links.findByLinkTypeAndObjectTypeAndDestination(RDStore.LINKTYPE_FOLLOWS,Subscription.class.name,subscriptionInstance.id)}"/>
+            <g:set var="previousSubscriptions" value="${Links.findByLinkTypeAndDestination(RDStore.LINKTYPE_FOLLOWS,com.k_int.kbplus.GenericOIDService.getOID(subscriptionInstance))}"/>
```
Comment (Moetez16) [inline app/grails-app/views/subscription/_actions.gsp:137]: com.k_int.kbplus.

File: app/grails-app/views/license/members.gsp [hunk 3]
```diff
@@ -110,7 +110,7 @@
         <g:each in="${validMemberLicenses}" status="i" var="row">
             <g:set var="lic" value="${row.license}"/>
             <%
-                LinkedHashMap<String, List> links = navigationGenerationService.generateNavigation(License.class.name, lic.id)
+                LinkedHashMap<String, List> links = linksGenerationService.generateNavigation(com.k_int.kbplus.GenericOIDService.getOID(lic))
```
Comment (Moetez16) [inline app/grails-app/views/license/members.gsp:113]: com.k_int.kbplus

File: app/grails-app/views/license/linkedSubs.gsp [hunk 4]
```diff
@@ -142,7 +137,7 @@
                 <g:set var="sub" value="${row}"/>
             </g:else>
             <%
-                LinkedHashMap<String, List> links = navigationGenerationService.generateNavigation(Subscription.class.name, sub.id)
+                LinkedHashMap<String, List> links = linksGenerationService.generateNavigation(com.k_int.kbplus.GenericOIDService.getOID(sub))
```
Comment (Moetez16) [inline app/grails-app/views/license/linkedSubs.gsp:140]: com.k_int.kbplus

Comment (croqueGrec09) [reply app/grails-app/views/license/linkedSubs.gsp]: Changed in bee15ab

File: app/grails-app/views/license/members.gsp (hunk 3 above)
Comment (croqueGrec09) [reply app/grails-app/views/license/members.gsp]: Changed in bee15ab

File: app/grails-app/views/subscription/_actions.gsp (hunk 2 above)
Comment (croqueGrec09) [reply app/grails-app/views/subscription/_actions.gsp]: Changed in bee15ab

File: app/grails-app/views/subscription/members.gsp (hunk 1 above)
Comment (croqueGrec09) [reply app/grails-app/views/subscription/members.gsp]: Changed in bee15ab

**Pull request merged**